src/window.rs

```diff
@@ -1035,6 +1027,35 @@ unsafe impl raw_window_handle::HasRawWindowHandle for Window {
     }
 }
 
+/// The behavior of cursor grabbing.
+///
+/// To grab cursor [`Window::set_cursor_grab_mode`] should be used.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum CursorGrabMode {
+    /// No grabbing of the cursor is performed.
+    None,
+
+    /// There's no guarantee that the cursor will be hidden. You should hide it by yourself if you
+    /// want so.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **macOS:** Not implemented.
```

Comment:
The macOS backend always returns `ExternalError::NotSupported` in this case. This line makes it seem like it *might* not do so.